spec/rubocop/cop/lint/rand_one_spec.rb

```diff
@@ -0,0 +1,100 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Lint::RandOne do
```

Comment:
I think the tests can be cleaned up by using `shared_examples` and `it_behaves_like` to change the variable being passed to the source.

```ruby
shared_examples 'offenses' do |num|
  let(:source) { "rand #{num}" }

  it 'registers an offense' do
    ...
  end
end

it_behaves_like 'offenses', 1
it_behaves_like 'offenses', -1
it_behaves_like 'offenses', 1.0
it_behaves_like 'offenses', -1.0
```